FlorenceBML/BMLacq756/BMLacq756.xml

```diff
@@ -29,7 +29,134 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                             <idno>Marrassini ms. 19</idno>
                         </altIdentifier>
                     </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <locus from="1a1" to="1a34"/>
+                            <title ref="LIT000000">Prayer against Bāryā, Tǝgrǝtyā and Belsibā</title>
+                            <note>Written mostly in Gǝʿǝz and partly in Amharic with invocations of magical names, mostly unintelligible.</note>
+                            <incipit xml:lang="gez"><gap reason="illegible" unit="lines" quantity="3"/> ጸሎት፡ በእንተ፡ ባርያ፡ ወትግርትያ፡ ቤልሲባ፡</incipit>
+                            <explicit xml:lang="gez">ለሰይጣና<gap reason="ellipsis" unit="chars" quantity="2"/> 
+                                አስማቲከ፡ <gap reason="illegible" unit="chars" quantity="4"/> ለአመትከ፡ </explicit>
```

Comment:
add `<textLang mainLang="gez" otherLangs="am"></textLang>` (also below, with the correct language)

Reply:
The information is taken from Marrassini and I can confirm, that the first prayer at least has one Amharic word (ነው), however, the incipit and the explicit do not contain any Amharic words.

Reply:
In any case, the element textLang  should be used regardless, even if only Geez is used

Reply:
I happened to identify features of Amharic in ms_i1, ms_i2 and ms_i7. Is it correct to encode the phrase individually as for the following: 
```
<msItem xml:id="ms_i7">
                            <locus from="1b160" to="1b189"/>
                            <title>Prayer against Šatolāy and Šatolāwit</title>
                            <textLang mainLang="gez" otherLangs="am"></textLang>
                            <note>Prayer against <term key="demon">Šatolāy</term> and <term key="demon">Šatolāwit</term> for healing from 
                                vaginal bleeding and for protection of newborn babies through the invocation of magic names
                                and through the binding of demons with the bonds of the Father, Son and Holy Spirit similar to 
                                <ref type="work" corresp="LIT6845PPSatolay"/>.</note>
                            <incipit xml:lang="gez">ጸሎት፡ በእንተ፡ <term key="demon">ሸቶላይ፡</term> ወ<term key="demon">ሸቶላዊት፡</term> አጋንንት፡ 
                                ዘትቀትሎ፡ ሕፃናተ፡ ወታወሕዙ፡ ደመ፡ አመሐኩክሙ፡ ወአውገዝኩከሙ፡ ወአርሰርኩክሙ፡</incipit>
                            <explicit xml:lang="gez">ደፈር፡ በሎንቃር፡ ፈውስ፡ ወአሕይዋ፡ ለአመትከ፡ 
                                <hi rend="rubric"><persName ref="PRS14227WalattaSellase">ወልተ፡ ሥላሴ፡</persName></hi>እመ፡ አልቦ፡ ነገር፡ ዘይሰአኖ፡ 
                                ለእግአብሔር፡</explicit>
                        </msItem>
```